.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+<!-- A brief description of the PR here. -->
+
+## Adding a new package?
+
+If this pull request adds a new package:
+
+- [ ] The package's schema URL is correct.
+- [ ] The package metadata file, if present, contains: 
+  - [ ] a supported category (one of `Cloud`, `Infrastructure`, `Network`, `Database`, `Monitoring`, or `Utility`).
+  - [ ] a description that explains what the package does.
+  - [ ] a valid logo URL that points to a PNG whose dimensions conform to the others in this repo (e.g., 100x100).
+  - [ ] a version number prefixed with `v` that corresponds with a valid GitHub release.
```

Comment:
[ ] The provider correctly identifies the source of creation being "Pulumi" (rather than TF)

@cnunciato You are welcome to re-word this, however I do want to make sure we get this right as many partners judge us by "impact" and this is how their dashboards light up.

Reply:
Specifically, this is the `User-Agent` string.